### PR TITLE
refactor: output flag code duplication

### PR DESCRIPTION
**Pull request by royhadad**

### CODE REVIEW

File: bl/evaluation/output_flag.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package evaluation
+
+var FormattedOutputOptions = []string{"yaml", "json", "xml", "JUnit"}
```
Comment (shmuelsa) [inline bl/evaluation/output_flag.go:3]: you done it right.

File: cmd/test/main_test.go [hunk 2]
```diff
@@ -814,15 +814,15 @@ func executeTestCommand(ctx *TestCommandContext, args []string) error {
 
 func test_testCommand_output_flags_validation(t *testing.T, ctx *TestCommandContext) {
 
-	validOutputValues := [5]string{"simple", "json", "yaml", "xml", "JUnit"}
+	validOutputValues := [...]string{"", "simple", "json", "yaml", "xml", "JUnit"}
```
Comment (shmuelsa) [inline cmd/test/main_test.go:817]: [...]


Alt Text

Comment (shmuelsa) [reply cmd/test/main_test.go:817]: `"" ` - is for empty case right?

**Review by shmuelsa — APPROVED** [review]

File: cmd/test/main_test.go (hunk 2 above)
Comment (royhadad) [reply cmd/test/main_test.go:817]: yep!

**Pull request merged**